lib/main.js

```diff
@@ -68,32 +79,32 @@ HTMLUglify.prototype.populateLookups = function(type, value, pointer, lookups) {
 };
 
 HTMLUglify.prototype.rewriteElements = function($, lookups) {
-  var self = this;
+  var _this = this;
 
   lookups = lookups || {};
 
   $('*[id]').each(function() {
-    self.pointerizeIdAndFor('id', $(this), lookups);
+    _this.pointerizeIdAndFor('id', $(this), lookups);
   });
 
   $('*[for]').each(function() {
-    self.pointerizeIdAndFor('for', $(this), lookups);
+    _this.pointerizeIdAndFor('for', $(this), lookups);
   });
 
   $('*[class]').each(function() {
-    self.pointerizeClass($(this), lookups);
+    _this.pointerizeClass($(this), lookups);
   });
 
```

Comment:
Sure, but nothing in the library is async under the hood so it wouldn't impact anything.